### ISSUE TITLE
The should be resolution limits

### ISSUE THREAD

**Issue by PixelVision8** [PixelVision8/GameCreator#17]
Right now you can increase the resolution with no limit. Perhaps there should be a limit at QVGA which is 320 x 240? [Wikipedia states](https://en.wikipedia.org/wiki/Third_generation_of_video_game_consoles) that 3rd gen systems didn't go above CGA resolution at 320 x 200. While 480p resolution does work, it appears to be very slow.

Need to address this in PV8 SDK and the Game Creator.

**Comment by LoneStranger** [comment]
I disagree about the limits. I think that the programmer should be able to determine which classic/fantasy limits they want to carry forward into their design.
